src/ui/form.js

```diff
@@ -43,13 +44,14 @@ export const form = function () {
     userObject.profession = inputUserProfession.value;
     userObject.tableNumber = inputTestNumber.value;
   };
-  inputDataForm.addEventListener("submit", function (e) {
+  inputDataForm.addEventListener('submit', function (e) {
     e.preventDefault();
     submitModalForm();
     closeModalForm();
     updateUserObject();
-    console.log(userObject);
+    table(shuffledArray(getBlackRedArr(Number(userObject.tableNumber))));
```

Comment:
haha, that's a bit too much, better to write a couple of lines declaring new variables, but let's leave it for now